ops/manifests/manifest-stable.yaml

```diff
@@ -4,7 +4,7 @@ applications:
   buildpacks:
     - python_buildpack
   path: ../../src
-  instances: 1
+  instances: 2
```

Comment:
Does this make the application have two instances? if so I think that is covered in another ticket that you may want to link to this PR with the resolves keyword. Also should we have the same 2 instances on staging so staging is as close to stable as possible?

Reply:
Good point, I'll mention the other ticket. And I can make the change to staging too, I think that sounds good.